modules/adding-to-a-project.adoc

```diff
@@ -11,5 +11,7 @@
 
 . Select a project.
 
-. In the upper right-hand corner of the *Project Status* menu, click *Add*, then
-choose from the provided options.
+. Click the *Workloads* tab.
+
+. Click *Import YAML* or *add other content*, then choose from the provided
```

Comment:
I'd suggest steering users to the Developer perspective where there are more options for adding content. (Select "Developer" from the context selector at the top of the nav, then "+Add")

For any existing topics that reference the console nav, we probably need to a step to switch to the Administrator perspective in case they're not already there.

cc @christianvogt